lib/nexpose/connection.rb

```diff
@@ -44,20 +44,29 @@ class Connection
     # The last XML response received by this object, useful for debugging.
     attr_reader :response_xml
 
+    # Enforcement of certificate validation to a CA
+    attr_reader :allow_untrusted
```

Comment:
We may be doing a 6.0 release soon anyway (other breaking changes) so it's not a big deal to me either way. You could even do separate branches for non-breaking and breaking changes if you are so inclined.